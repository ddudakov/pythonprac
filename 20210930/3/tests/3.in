1,2,0
5,7,0
7,7,0
1,8,9
1,7,4
2,5,3
