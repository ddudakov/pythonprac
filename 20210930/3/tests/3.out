[3, 22, 17]
[12, 89, 73]
[14, 105, 91]
